verilog/formatting/formatter_test.cc

```diff
@@ -878,8 +878,38 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
      "output reg yyy //c2\n"
      " ) ;endmodule:foo\n",
      "module foo (\n"
-     "    input  wire x,  //c1\n"   // aligned
-     "    output reg  yyy  //c2\n"  // trailing comments not aligned
+     "    input  wire x,   //c1\n"  // aligned
+     "    output reg  yyy  //c2\n"  // aligned
+     ");\n"
+     "endmodule : foo\n"},
+    {"module foo(  input wire x  ,/* c1 */\n"
+     "output reg yyy /* c2 */\n"
+     " ) ;endmodule:foo\n",
+     "module foo (\n"
+     "    input  wire x,   /* c1 */\n"  // aligned
+     "    output reg  yyy  /* c2 */\n"  // aligned
+     ");\n"
+     "endmodule : foo\n"},
+    {"module foo(  input wire x  ,/* c1\n"
+     "c2\n"
+     "c3 */\n"
+     "output reg yyy /* c4 */\n"
+     " ) ;endmodule:foo\n",
+     "module foo (\n"
+     "    input  wire x,   /* c1\n"
+     "c2\n"
+     "c3 */\n"  // TODO: align multiline comments
+     "    output reg  yyy  /* c4 */\n"
+     ");\n"
+     "endmodule : foo\n"},
+    {"module foo(  input wire x  ,/* c1 */\n"
+     "output reg yyy,\n"
+     "output z // c2\n"
+     " ) ;endmodule:foo\n",
+     "module foo (\n"
+     "    input  wire x,    /* c1 */\n"  // aligned
+     "    output reg  yyy,\n"
+     "    output      z     // c2\n"  // aligned
      ");\n"
```

Comment:
Maybe also adding a test with a bit-width mixed in ?
```systemverilog
  input logic [4:0] foo,  // comment
  input logic bar, // comment
```

Reply:
Added test.